### PR TITLE
Feature/quocdoan final pj/homeheader

### PR DESCRIPTION
**Pull request by at-quocdoan**
Update gem file log

### CODE REVIEW

**Comment by at-lamle** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controller/Home/Header/HomeHeaderViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+//  MyApp
+//
+//  Created by PCI0020 on 2/26/19.
+//  Copyright © 2019 Asian Tech Co., Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class HomeHeaderViewModel {
+
```
Comment (at-lamle) [inline MyApp/View/Controller/Home/Header/HomeHeaderViewModel.swift]: WARNING - Lines should not have trailing whitespace.

File: MyApp/View/Controller/Home/Header/HomeHeaderView.swift [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    var viewModel: HomeHeaderViewModel? {
+        didSet {
+            updateHeader()
+        }
+    }
+
+    // MARK: - Outlet
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var moreButton: Button!
+
```
Comment (at-lamle) [inline MyApp/View/Controller/Home/Header/HomeHeaderView.swift]: WARNING - Lines should not have trailing whitespace.

File: MyApp/View/Controller/Home/Header/HomeHeaderView.swift [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    var viewModel: HomeHeaderViewModel? {
+        didSet {
+            updateHeader()
+        }
+    }
+
+    // MARK: - Outlet
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var moreButton: Button!
+    // MARK: - Line cycle
```
Comment (at-lamle) [inline MyApp/View/Controller/Home/Header/HomeHeaderView.swift]: WARNING - Lines should not have trailing whitespace.

**Comment by at-lamle** [comment]
Total linter violations in pull request: 3
Serious: 0
Warnings: 3

**Comment by at-lamle** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controller/Home/Header/HomeHeaderView.swift (hunk 3 above)
Comment (at-quocdoan) [reply MyApp/View/Controller/Home/Header/HomeHeaderView.swift]: Em format code rồi nhưng nó không outdated file này a ạ.

**Comment by at-quocdoan** [comment]
@at-lamle @at-bienle Please review this PR again. Thanks!

**Pull request merged**